homeassistant/components/recorder/websocket_api.py

```diff
@@ -67,6 +68,27 @@ def ws_clear_statistics(
     connection.send_result(msg["id"])
 
 
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "recorder/get_statistics_metadata",
+        vol.Optional("statistic_ids"): [str],
+        vol.Optional("statistic_type"): vol.Any("sum", "mean"),
```

Comment:
Can we remove this parameter, it doesn't seem to be used by frontend PR https://github.com/home-assistant/frontend/pull/11892 ?

Reply:
`list_statistic_ids` needs `statistic_type` as it is used by `ha-statistic-picker.ts`. I added here it as well as maybe it might be useful in the future.

Reply:
Let's add it when frontend actually needs it, not now 👍